### ISSUE TITLE
LOLv2-Real指标出入很大

### ISSUE THREAD

**Issue by Cynicarlos** [LowLevelAI/GLARE#4]
作者您好，请问为什么在LOLv2-Real上的指标比论文中的高很多？如下：
![image](https://github.com/user-attachments/assets/6ed9fd08-55fd-42d7-a637-ebd6de8d3174)

**Comment by movingforward100** [comment]
您好！pretrained weights 是在LOL training set上训练得到的，而LOLv2-Real test set 内的数据全部包含在LOL training set内，因而指标很高。在LOLv2-Real training set上预训练的模型会在近期公开。如果有帮助，请帮忙点一下star并考虑引用论文，感谢关注~
